### DISCUSSION TITLE
Unclear website and documentation

### DISCUSSION THREAD

**Discussion by Anchakor** [ArchiveBox/ArchiveBox#1079] [General]
I've looked through the website and the wiki and couldn't find this info.
I think I have a quite common use case - I want archival of a single good snapshot of websites/documents from the internet.
When visiting the website as potential user I'd like to know:

1) Does ArchiveBox save just one good snapshot of the URL or many (like archive.org Wayback Machine), every time the URL is added? This is quite unclear on the website - one of the first sentences is "It saves *snapshots* of the URLs you feed it". Only much deeper you can read about "Saving Multiple Snapshots of a Single URL".

2) Does ArchiveBox allow fetching from services like archive.org Wayback Machine or archive.is if the URL is dead (4XX/5XX responses)?

3) How does one continue fetching the pending snapshots? I loaded thousands of URLs from bookmarks and rebooted machine before most were fetched. Should just putting up the HTTP server with `docker-compose up` resume the fetching? I found there is a `archivebox update` command which might do it, but it's not documented on the places I'd expect it to be.

**Comment by pirate** [comment]
1. You already found it, see https://github.com/ArchiveBox/ArchiveBox#saving-multiple-snapshots-of-a-single-url
2. No, see https://github.com/ArchiveBox/ArchiveBox/issues/160
3. `archivebox update`, or select them in the UI and click "Update" https://github.com/ArchiveBox/ArchiveBox/wiki/Usage#explanation-of-buttons-in-the-web-ui---admin-snapshots-list
